optional_plugins/golang/MANIFEST.in

```diff
@@ -1 +1 @@
-include VERSION
+include VERSION README.rst
```

Comment:
You added a very good description of why this change is needed in this commit, but I am missing the same thing in the other commits. Can you please copy/paste this to the other commits. IMO, it will increase commit history readability.